nginx/conf.d/app.conf

```diff
@@ -0,0 +1,6 @@
+server {
+  listen 80;
+  index index.js;
+  server_name _;
+  root /usr/share/nginx/html;
```

Comment:
Looking at the docs at https://hub.docker.com/_/nginx/ , I think we should just be able to do a `COPY --from=node app/out /usr/share/nginx/html`, and not require any configuration?

Reply:
ah ok. so you don't think we'll need this file?

Reply:
Yeah, I think we can make it much simpler and remove this file and the dockerfile line that copies this folder.